public/views/payment.html

```diff
@@ -1,7 +1,11 @@
-<form stripe-form="stripeCallback"> 
+<div ng-controller="DonationCtrl">
```

Comment:
You can alias the controller in ui-router and then you don't have to do it in-line here. Ask me if you don't know how to.